VALIDATION_GUIDE.md

```
@@ -0,0 +1,358 @@
+### 1. Python MLX Installation
+
+```bash
+cd /Users/arosboro/your_ai
+source venv/bin/activate  # or create new venv
+pip install mlx mlx-lm transformers
+```
+
+### 2. Model Export (TODO - Needs Implementation)
+
+The Rust trainer saved checkpoints as JSON:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)